python/py-selenium/Portfile

```diff
@@ -1,30 +1,33 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem      1.0
-PortGroup       python 1.0
+PortSystem          1.0
+PortGroup           python 1.0
 
-name            py-selenium
-set real_name   selenium
-version         3.141.0
-python.versions 27
-license         Apache-2
-maintainers     {dstrubbe @dstrubbe} openmaintainer
-platforms       darwin
-description     Python language binding for Selenium Remote Control
+name                py-selenium
+set real_name       selenium
+version             3.141.0
+license             Apache-2
+maintainers         {dstrubbe @dstrubbe} openmaintainer
+platforms           darwin
 
+description         Python language bindings for Selenium WebDriver
 long_description \
-    Selenium Python Client Driver is a Python language binding for \
-    Selenium Remote Control (version 1.0 and 2.0). Currently the remote protocol, \
-    Firefox and Chrome for Selenium 2.0 are supported, as well as the \
-    Selenium 1.0 bindings. As work will progresses we'll add more "native" drivers.
+    The selenium package is used to automate web browser interaction \
+    from Python. Several browsers/drivers are supported (Firefox, \
+    Chrome, Internet Explorer), as well as the Remote protocol.
 
-homepage            https://code.google.com/p/selenium/
+homepage            https://www.seleniumhq.org/
 master_sites        pypi:s/${real_name}/
 distname            ${real_name}-${version}
+
+# Support python versions
```

Comment:
either you mean "upstream-supported Python versions" (and should say so) or it's a useless comment as these are obviously the version we support

Reply:
Done thx.